internal/osbuildexecutor/runner-impl-aws-ec2.go

```diff
@@ -53,52 +95,133 @@ func (ec2e *awsEC2Executor) RunOSBuild(manifest []byte, store, outputDirectory s
 		}
 	}()
 
-	logrus.Info("Spinning up jobsite manager")
-	args := []string{
-		"--builder-host",
-		*si.Instance.PrivateIpAddress,
-		"--store",
-		store,
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*10)
+	defer cancel()
+	if !waitForSI(ctx, *si.Instance.PrivateIpAddress) {
+		return nil, err
 	}
+	controlFile, err := os.Open(filepath.Join("/var/tmp/osbuild-composer", "control.json"))
```

Comment:
I'm always a bit nervous about using hardcoded paths like this, if this code moves and two instances of it run at the same time this becomes racy and they will (potentially) override each others files. Maybe worth creating a tempdir here using `os.MkdirTemp()`?